osgp/platform/osgp-core/src/main/java/org/opensmartgridplatform/core/infra/messaging/CoreLogItemJsonMessageCreator.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2017 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.core.infra.messaging;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import javax.jms.Session;
+import javax.jms.TextMessage;
+import org.opensmartgridplatform.shared.infra.jms.Constants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CoreLogItemJsonMessageCreator {
```

Comment:
Why apply different styles of message creator classes for similar use cases with message senders?
The implementation choices with the `LoggingJsonMessageCreator` where the class named something ending with `MessageCreator` actually implements the `MessageCreator` interface looks cleaner than the choices made here.